2022/Q2/artifacts/pos-model/Staking.tla

```diff
@@ -14,29 +14,39 @@
 EXTENDS Integers, Apalache, Staking_typedefs
 
 CONSTANTS
-    \* Set of all addresses on Cosmos.
+    \* Set of all user addresses.
     \* @type: Set(ADDR);
     UserAddrs,
 
+    \* Set of all validator addresses.
+    \* ValidatorAddrs must be a subset of UserAddrs
+    \* @type: Set(ADDR);
+    ValidatorAddrs,
+
     \* @type: Int;
     PipelineLength,
 
     \* @type: Int;
     UnbondingLength
 
 VARIABLES
-    \* Coin balance for every Cosmos account.
+    \* Token balance for every account.
     \*
     \* @type: BALANCE;
     balanceOf,
-    \* Balance of unbonded coins that cannot be used during the bonding period.
+    \* Balance of unbonded tokens that cannot be used during the bonding period.
     \*
     \* @type: EPOCHED;
     unbonded,
-    \* Coins that are delegated to Validator.
+    \* Token that are delegated to Validator.
```

Comment:
Minor: constant "Validator" no longer exists. Please search "Validator" in comments and adapt them if needed